src/app/archive/archive_lib/processor.ml

```diff
@@ -98,8 +98,8 @@ module User_command = struct
     ; fee_payer_id: int
     ; source_id: int
     ; receiver_id: int
-    ; fee_token: string
-    ; token: string
+    ; fee_token: int
```

Comment:
`int64` would be better here (I think we can serialise that through Caqti?)

Reply:
I'll leave this for now since we're planning on fixing it with the custom uint64 serialize/deserialize